packages/lisk-transactions/src/utils/get_transaction_bytes.ts

```diff
@@ -229,9 +229,7 @@ export const getTransactionBytes = (transaction: TransactionJSON): Buffer => {
 	if (amountBigNum.lt(0)) {
 		throw new Error('Transaction amount must not be negative.');
 	}
-	// BUG in browserify-bignum prevents us using `.gt` directly.
-	// See https://github.com/bored-engineer/browserify-bignum/pull/2
-	if (amountBigNum.gte(new BigNum(MAX_TRANSACTION_AMOUNT).add(1))) {
+	if (amountBigNum.gte(new BigNum(MAX_TRANSACTION_AMOUNT))) {
```

Comment:
this bug is not an issue in forked version? if so should we use `gt` instead of `gte`

Reply:
good catch!